nebu/cli/get.py

```diff
@@ -175,7 +175,9 @@ def _write_node(node, base_url, out_dir, book_tree=False, get_resources=False,
     resp = requests.get('{}/contents/{}'.format(base_url, node['id']))
     if resp:  # Subcollections cannot (yet) be fetched directly
         metadata = resp.json()
-        resources = {r['filename']: r for r in metadata['resources']}
+        # skip auto-generated index.cnxml.html
```

Comment:
I think that the background info you provided in the PR's description would fit great here! And expanding your code would make it crystal clear that we're modifying resources to skip `index.cnxml.html` like so:

```suggestion
        resources = metadata['resources']
        
        """cnxml html files should not be edited nor published (they are
        generated and regenerated in-place in the database), so let's just
        not download them at all.
        """
        for res in resources:
            # skip index.cnxml.html
            if res['filename'] != 'index.cnxml.html':
                resources = { res['filename']: res }
```

Reply:
Oh, also, you might want to or have to modify line 198:

`-        gen_resources_sha1_cache(write_dir, metadata['resources'])`
`+        gen_resources_sha1_cache(write_dir, resources)`